Nos/Views/Components/MastodonBadgeView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+/// A view that displays a badge with a mastodon icon.
+///
+///
+/// This Text was implemented to be re-used in the Profile screen.
```

Comment:
A couple things I'd suggest:

1. As stated above, we probably want this to be ActivityPub instead of Mastodon.
2. I'm not sure what the last part of the doc comment is telling me. Could we remove or clarify? I guess what I'm not understanding is the statement that it's to be reused -- seems like something that's true for all Views.